graphics/GLC_lib/Portfile

```
@@ -37,7 +38,12 @@ configure.args      GLC_lib.pro
 
 # https://github.com/laumaya/GLC_lib/issues/30
 compiler.cxx_standard 2003
-configure.cxxflags-append -std=c++03
+
+if {${configure.compiler} in [list apple-gcc-4.0 apple-gcc-4.2 gcc-4.0 gcc-4.1 gcc-4.2 llvm-gcc-4.2]} {
+    configure.cxxflags-append -std=c++98
+} else {
+    configure.cxxflags-append -std=c++03
+}
```

Since there is apparently no need for C++03 rather than C++98, I would prefer to always use `-std=c++98` and set `compiler.cxx_standard` to `1998`.

```suggestion
configure.cxxflags-append -std=c++98
```